src/java/org/apache/cassandra/concurrent/SEPExecutor.java

```diff
@@ -70,7 +70,7 @@ public class SEPExecutor implements LocalAwareExecutorPlus, SEPExecutorMBean
     SEPExecutor(SharedExecutorPool pool, int maximumPoolSize, MaximumPoolSizeListener maximumPoolSizeListener, String jmxPath, String name)
     {
         this.pool = pool;
-        this.name = name;
+        this.name = NamedThreadFactory.globalPrefix() + name;
```

Comment:
this only impacts jvm-dtest.  We added this prefix to make the logs easier to read, the shared stages were missing this so was hard to see what `ReadStage` was the coming from which node